src/commands/files_delete.rs

```diff
@@ -0,0 +1,55 @@
+use std::collections::HashSet;
+
+use anyhow::Result;
+use clap::{Arg, ArgMatches, Command};
+
+use crate::api::Api;
+use crate::config::Config;
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .about("Delete a release file.")
+        // Backward compatibility with `releases files <VERSION>` commands.
+        .arg(Arg::new("version").long("version").hide(true))
+        .arg(
+            Arg::new("names")
+                .value_name("NAMES")
+                .multiple_occurrences(true)
+                .help("Filenames to delete."),
+        )
+        .arg(
+            Arg::new("all")
+                .short('A')
+                .long("all")
+                .help("Delete all files."),
+        )
+}
+
+pub fn execute(matches: &ArgMatches) -> Result<()> {
+    let config = Config::current();
+    let release = config.get_release_with_legacy_fallback(matches)?;
+    let org = config.get_org(matches)?;
+    let project = config.get_project(matches).ok();
+    let api = Api::current();
+
+    if matches.is_present("all") {
+        if api.delete_release_files(&org, project.as_deref(), &release)? {
+            println!("All files deleted.");
```

Comment:
shouldn't we account for the error case when we don't delete all the files? Same with similar functions.

Reply:
We do account for it, as api returns result, which will throw error with a valid reason thanks to `?` at the end of the call.